### PR TITLE
Document experiments section

### PR DESCRIPTION
**Pull request by oshi98**

### CODE REVIEW

File: documentation/docs/guide/experiments/create-an-experiment.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+1. On the H2O LLM Studio left-navigation pane, click **Create experiment**. Alternatively, you can click **New experiment** in the kebab menu of the [View datasets](../view-dataset.md) page.
+
+2. Select the dataset you want to use to fine-tune an LLM model.
+
+3. Select the problem type.
+
+4. Provide a meaningful experiment name.
+
+5. Select the LLM Backbone, the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended types of foundation models, but you can also use [Hugging Face models](https://huggingface.co/models).
```
Comment (sherenem) [inline documentation/docs/guide/experiments/create-an-experiment.md:13]: Let's make the field names bold and type them just as they appear on the UI (i.e., "Problem Type", "Experiment Name" etc)

File: documentation/docs/get-started/llm-studio-flow.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+- To learn about supported data connectors and data format, see [Supported data connectors and format](../guide/data-connectors-format).
+- To learn about how to import a dataset to H2O LLM Studio, see [Import a dataset](../guide/import-dataset).
+- To learn about reviewing and editing a dataset, see [View and manage dataset](../guide/view-dataset.md)
+
+## Step 2: Create an experiment
+
+As the second step in the experiment flow, create an experiment using the imported dataset. H2O LLM Studio offers several hyperparameter settings that you can adjust for your experiment model. To ensure that your training process is effective, you may need to specify the hyperparameters like learning rate, batch size, and the number of epochs. H2O LLM Studio provides an overview of all the parameters you’ll need to specify for your experiment.
+
+- To learn about the settings available for creating an experiment, see [Experiment settings](../guide/experiments/experiment-settings.md).
+- To learn about creating a new experiment, see [Create an experiment](../guide/experiments/create-an-experiment.md)
```
Comment (sherenem) [inline documentation/docs/get-started/llm-studio-flow.md:26]: I would switch around the order that this is listed in. First let's direct them to creating an experiment. Experiment settings seems more like a reference doc for more info. WDYT?

File: documentation/docs/get-started/llm-studio-flow.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+
+## Step 3: Monitor an experiment
+
+As the third step in the experiment flow, monitor the launced experiment. H2O LLM Studio allows you to inspect your experiment (model) during and after model training. Simple interactive graphs in H2O LLM Studio allow you to understand the impact of selected hyperparameter values during and after model training.
```
Comment (sherenem) [inline documentation/docs/get-started/llm-studio-flow.md:30]:
```suggestion
As the third step in the experiment flow, monitor the launched experiment. H2O LLM Studio allows you to inspect your experiment (model) during and after model training. Simple interactive graphs in H2O LLM Studio allow you to understand the impact of selected hyperparameter values during and after model training. You can then adjust the hyperparameters to further optimize model performance. 
```

File: documentation/docs/get-started/llm-studio-flow.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+
+To learn about viewing and monitoring an experiment, see [View and manage experiments](../guide/experiments/view-an-experiment.md).
+
+## Step 4: Compare experiments
+
+The H2O LLM studio provides a useful feature called Compare experiments, that allows comparing various experiments and analyzing how different model parameters affect model performance. This feature is a powerful tool for fine-tuning your machine-learning models and ensuring they meet your desired performance metrics.
```
Comment (sherenem) [inline documentation/docs/get-started/llm-studio-flow.md:36]:
```suggestion
The H2O LLM studio provides a useful feature  that allows comparing various experiments and analyzing how different model parameters affect model performance. This feature is a powerful tool for fine-tuning your machine-learning models and ensuring they meet your desired performance metrics.
```

File: documentation/docs/get-started/llm-studio-flow.md [hunk 5]
```diff
@@ -1 +1,45 @@
...
+- [Step 2: Create an experiment](#)
+- [Step 3: Monitor an experiment](#)
+- [Step 4: Compare experiments](#)
+- [Step 5: Export a model to Hugging Face Hub](#)
+
+In the below sections, each step above, in turn, is summarized.
+
+## Step 1: Import a dataset
+
+As the first step in the experiment flow, import your dataset to H2O LLM Studio.
```
Comment (sherenem) [inline documentation/docs/get-started/llm-studio-flow.md:15]:
```suggestion
As the first step in the experiment flow, prep your data and import your dataset to H2O LLM Studio. 
```

File: documentation/docs/guide/experiments/create-an-experiment.md [hunk 6]
```diff
@@ -4,13 +4,13 @@ Follow the relevant steps below to create an experiment in H2O LLM Studio.
...
+2. Select the **Dataset** you want to use to fine-tune an LLM model.
 
-3. Select the problem type.
+3. Select the **Problem type**.
 
-4. Provide a meaningful experiment name.
+4. Provide a meaningful **Experiment name**.
 
-5. Select the LLM Backbone, the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended types of foundation models, but you can also use [Hugging Face models](https://huggingface.co/models). 
+5. Select the **LLM Backbone**, the foundation model that you continue training. H2O LLM Studio has a predefined list of recommended types of foundation models, but you can also use [Hugging Face models](https://huggingface.co/models).
```
Comment (sherenem) [inline documentation/docs/guide/experiments/create-an-experiment.md:13]: As you suggested, let's add "LLM Backbone" to the key terms and link off to it to explain the concept.

File: documentation/docs/guide/experiments/delete-experiment.png
Comment (sherenem) [inline documentation/docs/guide/experiments/delete-experiment.png:1]: Reminder to change the highlighter shape to make it consistent with other product docs (yellow square with rounded edges)

File: documentation/docs/get-started/llm-studio-flow.md (hunk 2 above)
Comment (oshi98) [reply documentation/docs/get-started/llm-studio-flow.md:26]: Yeah, that makes sense. Thank you.

**Review by sherenem — APPROVED** [review]

**Comment by sherenem** [comment]
LGTM, thanks for the good work @oshi98 !

**Pull request merged**